website/docs/language/functions/toset.mdx

```diff
@@ -9,7 +9,9 @@ description: The toset function converts a value to a set.
 
 Explicit type conversions are rarely necessary in Terraform because it will
 convert types automatically where required. Use the explicit type conversion
-functions only to normalize types returned in module outputs.
+functions only to normalize types returned in module outputs, or in the
```

Comment:
Hi @threemachines! Sorry for the lull here, but I'm picking this back up and had a small suggestion for a rephrase:
```suggestion
functions only to normalize types returned in module outputs. If you want to normalize module output types from `toset`, see the [`for_each` meta-argument](/language/meta-arguments/for_each#using-sets).
```

Reply:
Apologies for my delay here, thank you for your response! Upon a second glance, I mainly want to break this into two sentences to separate the thought out. How does this feel to you? 
```suggestion
functions only to normalize types returned in module outputs. You can also convert output types from `toset` with the [`for_each` meta-argument](/language/meta-arguments/for_each#using-sets).
```